config/locales/en.yml

```diff
@@ -2904,7 +2904,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   first: "First"
   previous: "Previous"
   last: "Last"
-
+  
```

Comment:
there are extra characters here.

Reply:
corrected

Reply:
I am sorry about the little problems but you have added extra spaces or a tab where it should be empty line
always have a look at https://github.com/openfoodfoundation/openfoodnetwork/pull/5540/files